src/app/components/details/release_details.rs

```diff
@@ -65,7 +65,7 @@ mod imp {
 }
 
 glib::wrapper! {
-    pub struct ReleaseDetailsWindow(ObjectSubclass<imp::ReleaseDetailsWindow>) @extends gtk::Widget, libadwaita::Window;
+       pub struct ReleaseDetailsWindow(ObjectSubclass<imp::ReleaseDetailsWindow>) @extends gtk::Widget, libadwaita::Window;
```

Comment:
What happened here?

Reply:
idk, feel free to revert !

Reply:
(unless rustfmt forces you to keep it that is)